packages/compass-field-store/src/stores/store.ts

```diff
@@ -1,57 +1,14 @@
 import { createStore } from 'redux';
-import reducer, { changeFields } from '../modules';
-import type { Schema } from 'mongodb-schema';
-import parseSchema from 'mongodb-schema';
-import type { AppRegistry, ActivateHelpers } from 'hadron-app-registry';
-import { schemaFieldsToAutocompleteItems } from '../modules/fields';
+import reducer from '../modules';
+import type { ActivateHelpers } from 'hadron-app-registry';
 import { FieldStoreContext } from './context';
 
 export function activatePlugin(
   _initialProps: unknown,
-  { globalAppRegistry }: { globalAppRegistry: AppRegistry },
-  { on, cleanup }: ActivateHelpers
+  _: unknown,
+  { cleanup }: ActivateHelpers
```

Comment:
You don't need to pass cleanup here if you are not setting up any subscriptions, so you can clean this up a bit if you want (but also see my other comment, you might want to set something here listening to connection manager)